src/product-service/handlers/getProductsList.ts

```diff
@@ -0,0 +1,19 @@
+import products from '../data.json'
+export const getProductsList = async () => {
+  try {
+    console.log("products:", products)
```

Comment:
Please remove all console.log in MRs (console.logs are for debug, in MR we are presented finalized version of code)